### PR TITLE
[Crane] Update to alpha11

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

**Review by JoseAlcerreca — APPROVED** [review]
lgtm but someone more familiar with transitions ptal

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/BaseUserInput.kt [hunk 1]
```diff
@@ -102,7 +102,7 @@ fun CraneEditableUserInput(
             textStyle = if (isHint()) {
                 captionTextStyle.copy(color = AmbientContentColor.current)
             } else {
-                MaterialTheme.typography.body1
+                MaterialTheme.typography.body1.copy(color = AmbientContentColor.current)
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply Crane/app/src/main/java/androidx/compose/samples/crane/base/BaseUserInput.kt:105]: There's no `textColor` param in the `BasicTextField` API 🤯

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/CraneDrawer.kt [hunk 2]
```diff
@@ -37,7 +37,10 @@ private val screens = listOf("Find Trips", "My Trips", "Saved Trips", "Price Ale
 @Composable
 fun CraneDrawer(modifier: Modifier = Modifier) {
     Column(modifier.fillMaxSize().padding(start = 24.dp, top = 48.dp)) {
-        Image(imageVector = vectorResource(id = R.drawable.ic_crane_drawer))
+        Image(
+            imageVector = vectorResource(id = R.drawable.ic_crane_drawer),
+            contentDescription = null
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply Crane/app/src/main/java/androidx/compose/samples/crane/base/CraneDrawer.kt:42]: Done thanks

File: Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt [hunk 3]
```diff
@@ -227,6 +203,42 @@ private fun DayStatusContainer(
     }
 }
 
+private fun LazyListScope.itemsCalendarMonth(
+    month: CalendarMonth,
+    onDayClicked: (CalendarDay, CalendarMonth) -> Unit
+) {
+    item {
+        MonthHeader(
+            modifier = Modifier.padding(horizontal = 30.dp),
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt:212]: 😄  fixed

File: Crane/app/src/main/java/androidx/compose/samples/crane/home/MainActivity.kt [hunk 4]
```diff
@@ -100,30 +99,25 @@ private fun MainContent(
...
-
-private val splashTransitionDefinition = transitionDefinition<SplashState> {
-    state(SplashState.Shown) {
-        this[splashAlphaKey] = 1f
-        this[contentAlphaKey] = 0f
-        this[contentTopPaddingKey] = 100.dp
+@Composable
+private fun getSplashTransition(
+    transitionState: MutableTransitionState<SplashState>
+): Triple<State<Float>, State<Float>, State<Dp>> {
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply Crane/app/src/main/java/androidx/compose/samples/crane/home/MainActivity.kt:105]: Fixed, thanks!

**Comment by JoseAlcerreca** [comment]
Can you migrate it to alpha11?

**Comment by manuelvicnt** [comment]
Updated, thanks

File: Crane/app/src/main/java/androidx/compose/samples/crane/base/BaseUserInput.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisbanes) [reply Crane/app/src/main/java/androidx/compose/samples/crane/base/BaseUserInput.kt:105]: Eugh, annoying

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**